packages/insomnia-app/app/ui/components/modals/request-group-settings-modal.js

```diff
@@ -0,0 +1,338 @@
+        keyMap={editorKeyMap}
+        placeholder="Write a description"
+        lineWrapping={editorLineWrapping}
+        handleRender={handleRender}
+        handleGetRenderContext={handleGetRenderContext}
+        nunjucksPowerUserMode={nunjucksPowerUserMode}
+        isVariableUncovered={isVariableUncovered}
+        defaultValue={requestGroup.description}
+        onChange={this._handleDescriptionChange}
+      />
+    ) : (
+      <button
+        onClick={this._handleAddDescription}
+        className="btn btn--outlined btn--super-duper-compact">
+        Add Description
+      </button>
+    );
+  }
+
+  _renderMoveCopy(): React.Node {
+    const { workspaces } = this.props;
+
+    const {
+      activeWorkspaceIdToCopyTo,
+      justMoved,
+      justCopied,
+      workspace,
+      requestGroup,
+    } = this.state;
+
+    if (!requestGroup) {
+      return null;
+    }
+
+    return (
+      <div className="form-row">
+        <div className="form-control form-control--outlined">
+          <label>
+            Move/Copy to Workspace
+            <HelpTooltip position="top" className="space-left">
+              Copy or move the current request group to a new workspace. It will be placed at the
+              root of the new workspace's folder structure.
+            </HelpTooltip>
+            <select
+              value={activeWorkspaceIdToCopyTo || '__NULL__'}
+              onChange={this._handleUpdateMoveCopyWorkspace}>
+              <option value="__NULL__">-- Select Workspace --</option>
+              {workspaces.map(w => {
+                if (workspace && workspace._id === w._id) {
+                  return null;
+                }
+
+                return (
+                  <option key={w._id} value={w._id}>
+                    {w.name}
+                  </option>
+                );
+              })}
+            </select>
+          </label>
+        </div>
+        <div className="form-control form-control--no-label width-auto">
+          <button
+            disabled={justCopied || !activeWorkspaceIdToCopyTo}
+            className="btn btn--clicky"
+            onClick={this._handleCopyToWorkspace}>
+            {justCopied ? 'Copied!' : 'Copy'}
+          </button>
+        </div>
+        <div className="form-control form-control--no-label width-auto">
+          <button
+            disabled={justMoved || !activeWorkspaceIdToCopyTo}
+            className="btn btn--clicky"
+            onClick={this._handleMoveToWorkspace}>
+            {justMoved ? 'Moved!' : 'Move'}
+          </button>
+        </div>
+      </div>
+    );
+  }
+
+  renderModalBody(): React.Node {
+    const { requestGroup } = this.state;
+
+    if (!requestGroup) {
+      return null;
+    }
+
+    return (
+      <div>
+        <div className="form-control form-control--outlined">
+          <label>
+            Name{' '}
+            <span className="txt-sm faint italic">(also rename by double-clicking in sidebar)</span>
```

Comment:
Double-clicking a folder just expands or collapses it 😆 

```suggestion
            Name
```